ckanext/canada/templates/internal/scheming/package/snippets/package_form.html

```diff
@@ -13,6 +13,11 @@
         <summary>
           {{_('Catalogue Metadata')}}
         </summary>
+        {% if dataset_type == 'info' %}
+          {% set m1 = _('Open Information Metadata Element Set') %}
+        {% else %}
+          {% set m1 = _('Open Data Metadata Element Set') %}
+        {% endif %}
```

Comment:
this is ok, but in general I would prefer to put these strings in the schema (so we don't have if/else cases littering templates)